<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Alkoholrechner</class>
 <widget class="QDialog" name="Alkoholrechner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alkoholrechner</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QGroupBox" name="gb_person">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>338</width>
     <height>136</height>
    </rect>
   </property>
   <property name="title">
    <string>Angabe zur Person</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="txt_geschlecht">
      <property name="text">
       <string>Geschlecht:</string>
      </property>
      <property name="buddy">
       <cstring>cbox_geschlecht</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="txt_alter">
      <property name="text">
       <string>Alter:</string>
      </property>
      <property name="buddy">
       <cstring>le_alter</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="cbox_geschlecht">
      <item>
       <property name="text">
        <string>weiblich</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>männlich</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="le_alter"/>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="txt_angabe_year">
      <property name="text">
       <string>Jahre</string>
      </property>
      <property name="buddy">
       <cstring>le_alter</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="txt_gewicht">
      <property name="text">
       <string>Gewicht:</string>
      </property>
      <property name="buddy">
       <cstring>le_gewicht</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="txt_grosse">
      <property name="text">
       <string>Größe:</string>
      </property>
      <property name="buddy">
       <cstring>le_grosse</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="le_gewicht"/>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="txt_angabe_kg">
      <property name="text">
       <string>kg</string>
      </property>
      <property name="buddy">
       <cstring>le_gewicht</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="le_grosse"/>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="txt_angabe_cm">
      <property name="text">
       <string>cm</string>
      </property>
      <property name="buddy">
       <cstring>le_grosse</cstring>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="gb_alkohol">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>331</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Angabe zum Konsum</string>
   </property>
   <widget class="QLabel" name="txt_zeitraum">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Zeitraum:</string>
    </property>
   </widget>
   <widget class="QLabel" name="txt_angabe_h">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>h</string>
    </property>
   </widget>
   <widget class="QLabel" name="txt_list_getranke">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Liste Getränke:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tabwid_getranke">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>301</width>
      <height>151</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>1</number>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>95</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <row/>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>% Vol.</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Menge in L</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>...</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>...</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>...</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="cmd_getr_hin">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Get. Hinzu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cmd_getr_del">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Get. Del</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_time">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_erg">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>331</width>
     <height>431</height>
    </rect>
   </property>
   <property name="title">
    <string>Ergebnis</string>
   </property>
   <widget class="QTextBrowser" name="txt_browser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>311</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="txt_erg">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ergebnis:</string>
    </property>
   </widget>
   <widget class="QLabel" name="txt_erg_promille">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="txt_gewicht_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Beschreibung:</string>
    </property>
    <property name="buddy">
     <cstring>txt_browser</cstring>
    </property>
   </widget>
   <widget class="QPushButton" name="cmd_ber">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Berechnen!</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>4</number>
    </property>
    <property name="value">
     <number>-1</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
